README.md

```diff
@@ -276,6 +276,42 @@ It supports the ArchEthic Cryptography rules which are:
   const { fee: fee } = await archethic.getTransactionFee(tx, "https://testnet.archethic.net")
   ```
 
+  ### Keychain / Wallet management
```

Comment:
It could be nice to explain how to use Keychain after it's creation.
for exemple how to use service to get an address (```Keychain.deriveAddress(services, index)```)

Reply:
👍